src/test/resources/features/refapp-2.x/12-condition/conditions.feature

```diff
@@ -1,43 +1,65 @@
 Feature: Patient's Conditions Managment
 
-  Background:
+  Background: 
     Given User clicks on Conditions from Patient dashboard
     Then System loads Manage Conditions Page
-
-  @selenium
-  @dashboard
-  Scenario Outline: Adding an active or inactive condition
     And User clicks on Add new condition
     Then System on Add New Condition Page
-    And User enters "<activity>" condition
-    And User clicks save
-    Then System loads Manage Conditions Page
-    Examples:
-      | activity |
-      | active   |
-      | inactive |
 
-  @dashboard
-  Scenario: Set first condition to inactive
+  @selenium
+  @condition
+  Scenario: Active condition management
+    # User story: Adding an active condition
+    And User enters active condition
+    And User clicks save
+    
+    # User story: Set first condition to inactive
     And User clicks on set inactive button
     Then System should move condition to inactive section
 
-  @dashboard
-  Scenario: Set first condition to active
+  @selenium
+  @condition
+  Scenario: Inactive condition management
+    # User story: Adding an active condition
+    And User enters inactive condition
+    And User clicks save
+    
+    # User story: Set first condition to active
     And User clicks on set active button
     Then System should move condition to active section
 
-  @dashboard
-  Scenario: Edit first active condition
+  @selenium @condition
```

Comment:
any reason for putting the two tags on the same line?

Reply:
sorry about that, pretty formatter did that. lemi correct that